backend/public-graph/graph/resolver.go

```diff
@@ -833,6 +833,12 @@ func (r *Resolver) HandleErrorAndGroup(ctx context.Context, errorObj *model.Erro
 			return nil, ErrNoisyError
 		}
 	}
+	if projectID == 3322 {
```

Comment:
unrelated change, but feel free to keep this. just want to check if this is still relevant.